packages/contentful/src/gp2/queries/reminder.queries.ts

```diff
@@ -65,5 +69,45 @@ export const FETCH_REMINDERS = gql`
         }
       }
     }
+    outputVersionCollection(where: $outputVersionFilter) {
+      items {
+        sys {
+          id
+          publishedAt
+        }
+        linkedFrom {
+          outputsCollection(limit: 1) {
+            items {
+              sys {
+                id
+              }
+              title
+              documentType
+              createdBy {
+                firstName
+                lastName
+              }
+              relatedEntitiesCollection(limit: 1) {
```

Comment:
I've noticed now reading the spreadsheet with the reminders instructions (https://docs.google.com/spreadsheets/d/1dDB2wboG6cS7FW3suPa7K2k9PCTCIZRcY8le31nKyUA/edit#gid=0)

<img width="1366" alt="Screenshot 2024-02-08 at 07 44 52" src="https://github.com/yldio/asap-hub/assets/16595804/7d237896-d046-4950-bc5d-2a86be120a52">

that there's an issue here and with the published outputs. We are fetching only the main entity not all of them and the reminder should be displayed to all of them 😅

Reply:
I've raised this with Tiffany. She's noted it down and will discuss with João when he's back.